cmd/ttn-lw-stack/commands/start.go

```diff
@@ -159,6 +160,7 @@ var startCommand = &cobra.Command{
 			config.NS.ApplicationUplinks = nsredis.NewApplicationUplinkQueue(redis.New(&redis.Config{
 				Redis:     config.Redis,
 				Namespace: []string{"ns", "application-uplinks"},
+				PoolSize:  math.MaxInt32, // Mitigate https://github.com/TheThingsNetwork/lorawan-stack/issues/1598
```

Comment:
It doesn't make a lot of sense to set the limit this high, considering that 10000 is Redis's default maximum. It seems AWS ElastiCache allows 65000, so that may be a good choice too.